.github/workflows/main.yml

```diff
@@ -101,3 +101,12 @@ jobs:
       - run: python3 -m pip install mypy==0.730
       - run: python3 -m mypy --strict --allow-untyped-calls --show-error-codes -p sdb
       - run: python3 -m mypy --strict --ignore-missing-imports --show-error-codes -p tests
+  #
+  # Verify that "shfmt" ran successfully against our shell scripts.
+  #
+  shfmt:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - run: ./.github/scripts/install-shfmt.sh
+      - run: /usr/local/bin/shfmt -d .github/scripts/
```

Comment:
Wouldn't it better to run it against the whole repository?

Reply:
I don't see why not. Making the change now

Reply:
Changed it for the whole sdb directory - I think this is worthwhile as I may introduce a few scripts outside of the `.github` directory while trying to get sdb on PyPI so that it can be installed through `pip`.